### PR TITLE
overview refactor

### PR DESCRIPTION
**Pull request by kauffj**
Closes https://github.com/lbryio/lbry.tech/issues/51

### CODE REVIEW

File: app/components/ecosystem/module-applications.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 "use strict";
 
+import { require as local } from "app-root-path";
```
Comment (NetOperatorWibby) [inline app/components/ecosystem/module-applications.js]: Were you not able to do `import markdown from "../markdown";`? That way of importing should work now.

File: documents/overview.md [hunk 2]
```diff
@@ -6,32 +6,31 @@ title: Overview
...
 
-That's the mission of LBRY.
-
 <MissionStatement/>
 
-That's a fancy sentence, so here's a plainer one: we just thought it'd be really damn cool if there was a system that made it easy to discover and distribute as much of the world's information as possible but was owned and controlled by no one.
+That's a fancy sentence, so here's a plain one: we thought it'd be damn cool if there was a system that made it easy to discover and distribute as much of the world's information as possible but was owned and controlled by no one.
 
-^1(#footnote-1)^ In the information theoretic sense, LBRY facilitates distribution of all data, whether it be a video or a spreadsheet.
+<sub>^1(#footnote-1)^ In the information theoretic sense, LBRY facilitates distribution of all data, whether it be a video or a spreadsheet.</sub>
```
Comment (NetOperatorWibby) [inline documents/overview.md]: Ah, this is where the `<sub>` issue comes from.

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (NetOperatorWibby) [inline .idea/codeStyles/Project.xml]: Do we want this `.idea` folder in the repo?

File: app/components/ecosystem/module-applications.js (hunk 1 above)
Comment (kauffj) [reply app/components/ecosystem/module-applications.js]: I just copy and pasted what was done elsewhere.

File: .idea/codeStyles/Project.xml (hunk 3 above)
Comment (kauffj) [reply .idea/codeStyles/Project.xml]: Absolutely not, my mistake. I thought I had this removed from the commit. I'm on a new computer and lost my global git ignore.

**Comment by tzarebczan** [comment]
What's the status of this PR?

**Comment by kauffj** [comment]
@NetOperatorWibby I fixed the import statements and removed the unnecessary files, merging.

**Pull request merged**